rest_framework/compat.py

```diff
@@ -169,6 +169,21 @@ def parse_header_parameters(line):
         }
 
 
+if django.VERSION >= (5, 0):
+    # Django 5.0+: use the stock ip_address_validators function
+    # Note: Before Django 5.0, ip_address_validators returns a tuple containing
+    #       1) the list of validators and 2) the error message. Starting from
+    #       Django 5.0 ip_address_validators only returns the list of validators
+    from django.core.validators import ip_address_validators
```

Comment:
The change that caused this issue is with the current Django `main` branch, not `5.0`, so this needs to be changed.

Reply:
Affected Django version is 5.1.